orgparse/node.py

```diff
@@ -717,26 +719,25 @@ def __unicode__(self):
         def __str__(self):
             return unicode(self).encode('utf-8')
 
-    def get_file_property(self, property):
+    # todo hmm, not sure if it really belongs here and not to OrgRootNode?
```

Comment:
Hi, I'm following your project and just saw that comment, maybe this is helpful: https://orgmode.org/Changes.html#org935ad43

> Property drawers are now allowed before the first headline.

> Org mode is moving more towards making things before the first headline behave just as if it was at outline level 0. Inheritance for properties will work also for this level. In other words: defining things in a property drawer before the first headline will make them "inheritable" for all headlines.

So imho it makes sense to move this to `OrgRootNode`

Reply:
Oh nice, thanks for a heads-up! I think what they mean are the actual properties drawers though? (as opposed to file properties file `#+TITLE:`). That would mean moving `_properties` field and the related parsing code to `OrgBaseNode`.

This todo, however, refers to file properties like `#+TITLE` or `#+FILETAGS`, so it kinda makes sense that only `OrgRootNode` has the corresponding methods. On the other hand, currently level 1+ nodes would just return `None`, so it's not too bad either.

Reply:
Ah yeah, makes sense. I was confused by "things before the first headline", i had the impression a file property `#+title` would be equivalent to a `:title:` in a property drawer. But your explanation seems more logical ;)